paths/job_templates/destroy.yaml

```diff
@@ -0,0 +1,44 @@
+---
+summary: Delete a job template
+description: Delete an existing job template.
+operationId: job_template/delete
+tags:
+- Job Templates
+parameters:
+- "$ref": "../../parameters.yaml#/X-PhraseApp-OTP"
+- "$ref": "../../parameters.yaml#/project_id"
+- "$ref": "../../parameters.yaml#/id"
+- description: specify the branch to use
+  example: my-feature-branch
+  name: branch
+  in: query
+  schema:
+    type: string
+responses:
+  '204':
+    "$ref": "../../responses.yaml#/204"
+  '400':
+    "$ref": "../../responses.yaml#/400"
+  '404':
+    "$ref": "../../responses.yaml#/404"
+  '429':
+    "$ref": "../../responses.yaml#/429"
+x-code-samples:
+- lang: Curl
+  source: |-
+    curl "https://api.phrase.com/v2/projects/:project_id/job_templates/:id" \
+      -u USERNAME_OR_ACCESS_TOKEN \
+      -X DELETE \
+      -d '{"branch":"my-feature-branch"}' \
+      -H 'Content-Type: application/json'
+- lang: CLI v2
+  source: |-
+    phrase job templates delete \
+    --project_id <project_id> \
+    --id <id> \
+    --branch my-feature-branch \
+    --access_token <token>
+- lang: CLI v1
```

Comment:
We don't support CLI v1 anymore, please remove it everywhere from those examples 🎉